python/data_structures/hashtable.py

```diff
@@ -1,12 +1,94 @@
+
+from data_structures.linked_list import LinkedList
+
 class Hashtable:
     """
     Put docstring here
     """
+    def __init__(self, size=1024):
+        self.size = size
+        self._buckets = [None] * size
+
+    def set(self, key, value):
+        '''
+        Sets item into hashtable by first calculating hash, then if it doesn't have anything in it, then instantiate a linked list to hold valuse in case of possible collision. 
```

Comment:
I like the docstrings! Recommend adding them to the methods below missing them as well. Could probably copy/paste the readme's comments for each method over.